src/main/java/optimus/Ui.java

```diff
@@ -0,0 +1,46 @@
+package optimus;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    Scanner scanner;
+    private static final String linebreak = "____________________________";
```

Comment:
Might want to change the name for convention?
```suggestion
    private static final String LINE_BREAK = "____________________________";
```